db/migrate/20151435234635_add_container_definition_security_contexts.rb

```diff
@@ -0,0 +1,13 @@
+class AddContainerDefinitionSecurityContexts < ActiveRecord::Migration
+  def change
+    add_column    :container_definitions, :privileged, :boolean
+    add_column    :container_definitions, :run_as_user, :bigint
+    add_column    :container_definitions, :se_linux_user, :string
+    add_column    :container_definitions, :se_linux_role, :string
+    add_column    :container_definitions, :se_linux_type, :string
+    add_column    :container_definitions, :se_linux_level, :string
+    add_column    :container_definitions, :run_as_non_root, :boolean
+    add_column    :container_definitions, :capabilities_add, :string
+    add_column    :container_definitions, :capabilities_drop, :string
```

Comment:
Can you describe what `capabilities_add` and `capabilities_drop` are?  I can't tell just by the column names what they're supposed to hold.

Might make sense to rename these if there's a better way to describe them...